### ISSUE TITLE
Razor Page completions shown for older projects

### ISSUE THREAD

**Issue by danroth27** [aspnet/Razor.VSCode#235]
In an ASP.NET Core 1.0 project:

![image](https://user-images.githubusercontent.com/1874516/48464763-768e9800-e795-11e8-9d20-9c5b4e9b7102.png)

**Comment by danroth27** [comment]
Looks like this only happens before the packages are restored for the project.

**Comment by NTaylorMullen** [comment]
Yup, that makes sense. We default to latest Razor until we know better 👍
